django_6pm/django_6pm/product/models.py

```diff
@@ -0,0 +1,47 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class Brand(models.Model):
+    name = models.CharField(max_length=250)
+    logo = models.URLField()
+
+
+class Category(models.Model):
+    name = models.CharField(max_length=250)
+    parent = models.ForeignKey("self", on_delete=models.SET_NULL, null=True, blank=True)
+
+
+class Product(models.Model):
+    class Gender(models.IntegerChoices):
+        WOMEN = 1, _("women")
+        MEN = 2, _("men")
+        UNISEX_ADULTS = 3, _("unisex-adults")
+        BOYS = 4, _("boys")
+        GIRLS = 5, _("girls")
+        UNISEX_KIDS = 6, _("unisex-kids")
+
+    name = models.TextField()
+    gender = models.IntegerField(choices=Gender.choices)
```

Comment:
`gender` should be CharField

Reply:
I was asked by Mubtada to make it enum as it helps with faster filteration